app/assets/javascripts/miq_application.js

```diff
@@ -1231,8 +1231,11 @@ function miqAccordSelect(e) {
     return false;
   }
 
-  var url = '/' + $('body').data('controller') + '/accordion_select?id=' + $(e.target).attr('id');
-  miqJqueryRequest(url, {beforeSend: true, complete: true});
+  // No need to load anything if only a single accordion is present
+  if ($('#accordion > .panel').length > 1) {
```

Comment:
@himdel I have a feeling that you might not like this approach here, but as we're going in the direction of components, this will be eventually moved into a controller/component around accordions.